### DISCUSSION TITLE
Question on Contribution - Addition of new unit types

### DISCUSSION THREAD

**Discussion by andorrax101** [unitsofmeasurement/unit-api#252] [Ideas]
Hello ! 
I wish to contribute to the unit-api repo and my first thought was to potentially add new units to the quantitiy package. 
I'm using unitsNet from C# (repo link here: https://github.com/angularsen/UnitsNet/tree/master/UnitsNet/CustomCode/Quantities) as my basis for this. It is used quite extensively in the dotNet world and I would like to bring over as many of these units as I can (with proper discussion and due process of course !). 

What's the best way to go about this ? Would I need to become a JCP member to begin contributing ? Or is it okay to create an issues ticket and raise PRs ?

**Comment by desruisseaux** [comment]
@keilw is the one who can tell about the process. But as a note about the content, the goal is not to put as many quantities as possible. The main guideline is that the quantity must be defined by a recognized standard organization, preferably an international organization (rather than a national organization) such as the [Bureau International des Poids et Mesures (BIPM)](https://www.bipm.org/) and the [International Organization for Standardization (ISO)](https://www.iso.org/). Other sources are generally avoided because there is no limit to the amount of quantities that could be added, with everyone having their own opinion of what is a "widely used" quantity.

**Comment by keilw** [comment]
As @desruisseaux mentioned, this repository is the wrong place for any additional quantity. 
If you found any quantity still missing from [SI Quantities](https://github.com/unitsofmeasurement/si-units/tree/master/quantity) that's defined by the BIPM catalog, then please file an issue under [SI Units](https://github.com/unitsofmeasurement/si-units/issues).
Other quantities may fit into [Systems Quantities](https://github.com/unitsofmeasurement/uom-systems/tree/master/quantity/src/main/java/systems/uom/quantity), please file any enhancement tickets under [UOM Systems](https://github.com/unitsofmeasurement/uom-systems/issues).
